### ISSUE TITLE
Describe sample depth relative to a core

### ISSUE THREAD

**Issue by dannymandel** [isamplesorg/metadata#35]
In GEOME, depth looks like this:

```
"maximumDepthInMeters": "12"
```

but it might be relative to something else, and our elevation field describes elevation relative to sea level.  How do we reconcile this and detect it in GEOME data?

**Comment by dannymandel** [comment]
We have one field in here:

https://github.com/isamplesorg/metadata/blob/main/iSamplesSchemaBasic0.2.json

and it's titled "elevation".

**Comment by smrgeoinfo** [comment]
Yup, the one field is the simple version; for now I was figuring we'd just have to put in whatever is in the source, 'verbatim'.  In good data, the elevation/depth should be associated (either at the record or dataset level) with a spatial reference system that specifies the vertical datum, units of measure, and positive direction.  Lacking that, we have to assume datum is 'mean sea level', and depth is measured positive down from that, elevation is measured positive up from that.

**Comment by dannymandel** [comment]
Assigning this one back to me to implement.

**Comment by dannymandel** [comment]
Thanks for the exaplanation @smrgeoinfo

**Comment by smrgeoinfo** [comment]
Close?

**Comment by dannymandel** [comment]
It's done from yours and @datadavev's perspective, but I haven't implemented it yet.  I'd prefer to keep open and assigned to me until I've written the code to implement it.
